CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+# Developer
+
+We use [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html) version [4.0.0](https://github.com/mapbox/mason/tree/master/scripts/clang-format) to consistently format the code base. The format is automatically checked via a Travis CI build as well. Run the following script locally to ensure formatting is ready to merge:
```

Comment:
We actually use version `4.0.1` currently. How about we remove the version mention here and instead say "Look inside https://github.com/mapbox/hpp-skel/blob/master/scripts/setup.sh#L7` to see the version in use.